CMakeLists.txt

```diff
@@ -73,9 +80,18 @@ if (${UNIX})
 
     set(PY_SHORT_VERSION "${PY_VERSION_MAJOR}${PY_VERSION_MINOR}")
 
-    find_package(Boost 1.54 REQUIRED COMPONENTS filesystem python-py${PY_SHORT_VERSION})
+    getLibVersion(libboost-python-dev BOOST_PYTHON_VER)
+    if ("${BOOST_PYTHON_VER}" STRGREATER "1.71")
```

Comment:
Different versions of libboost-python have their .so library for python3 named differently